CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## v1.20.0 (06/08/2024)
+
+**Common**
+* Updated links and filenames in the documentation.
+* Improved the CI scripts and pipelines.
+
+**olp-cpp-sdk-authentication**
+* Fixed the DateTime parser to respect DST timezones on iOS.
+* Fixed determining default credentials file absolute path on Windows for new distributions.
```

Comment:
I'm not sure I understand what this means. I think you fixed the mechanism that determines the absolute path of the credentials file on Windows, correct? When you say "for new distributions", do you mean new WINDOWS or new SDK?

Reply:
Yes, we fixed the mechanism. The old version of the mechanism didn't work on the new Windows distributions, Like Windows 11.

Reply:
OK, with that said, I propose to change this to: 
```suggestion
* Fixed the mechanism that determines the absolute path of the credentials file on new Windows versions, such as Windows 11.
```
I don't think I've seen the word "distribution" used in the Windows context so I went with "version".

Reply:
Done